drape_frontend/rule_drawer.cpp

```diff
@@ -76,13 +77,23 @@ double GetBuildingHeightInMeters(FeatureType & f)
 
 double GetBuildingMinHeightInMeters(FeatureType & f)
 {
-  auto const value = f.GetMetadata(feature::Metadata::FMD_MIN_HEIGHT);
-  if (value.empty())
-    return 0.0;
+  double minHeightInMeters = 0.0;
 
-  double minHeightInMeters;
-  if (!strings::to_double(value, minHeightInMeters))
-    minHeightInMeters = 0.0;
+  auto value = f.GetMetadata(feature::Metadata::FMD_MIN_HEIGHT);
+  if (!value.empty())
+  {
+    if (!strings::to_double(value, minHeightInMeters))
+      minHeightInMeters = 0.0;
+  }
+  else
+  {
+    value = f.GetMetadata(feature::Metadata::FMD_BUILDING_MIN_LEVEL);
+    if (!value.empty())
+    {
+      if (strings::to_double(value, minHeightInMeters))
+      minHeightInMeters *= kMetersPerLevel;
```

Comment:
Seems like better to process same as in height:
```
if (!strings::to_double(value, minHeightInMeters))
  minHeightInMeters = 0;
else
  minHeightInMeters *= kMetersPerLevel;
```
And use spaces for indent instead of tabs.